client/components/Toggle/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import classNames from 'classnames'
+import './style.scss'
+
+export default function Toggle({ value, onChange, readOnly }) {
+    const onClick = () => onChange({ target: { value: !!value } })
```

Comment:
I only renamed the file to .jsx, added readOnly state and unit tests, the remaining is the original functionality.

I can change this Toggle to send the new Toggle state if you wish?